hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionPlugin.java

```diff
@@ -32,8 +31,8 @@ private void createConvention(Project target) {
   }
 
   private void configureCompatibility(Project target) {
-    Property<JavaVersion> javaVersion = javaConventionExtension(target).compatibilityVersion;
-    javaPluginExtension(target).setTargetCompatibility(javaVersion);
+    JavaVersion javaVersion = javaConventionExtension(target).compatibilityVersion.get();
+    javaPluginExtension(target).setSourceCompatibility(javaVersion);
```

Comment:
Why are we setting source and target compatibility from the same value?

Reply:
Yes I thought in our usage at least we wouldn't need different. I can add another property if you feel such a case might come up

Reply:
Let us continue this on https://github.com/hypertrace/hypertrace-gradle-java-convention-plugin/pull/4